vacuum-advisor/bloat.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'VACUUM Advisor: Bloat in Postgres'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+In Postgres, bloat refers to a situation that the physical size of a table on
+disk is larger than its actual data size.
+
+## Why does bloat happen?
+
+In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify.
+VACUUM allows Postgres to reclaim these dead rows and reuse that space for
+future INSERTs or UPDATEs. Autovacuum is designed to perform VACUUM regularly in
+order to efficiently reuse space. This helps ensure that INSERTs and UPDATEs can
+use existing space instead of claiming new space. This helps prevent unnecessary
+growth of tables by utilizing existing space.
```

Comment:
```suggestion
order to efficiently reuse space. This helps prevent unnecessary
table growth.
```
I think this could be more compact. Those extra words aren't saying anything.